docs/variables.md

```diff
@@ -113,3 +113,36 @@ variable via `resources.inputs.<resourceName>.<variableName>` or
 | `name` | The name of the resource. |
 | `type` | Type value of `"cloudEvent"`. |
 | `target-uri` | The URI to hit with cloud event payloads. |
+
+## Variables that can be substituted in a CRD
+
+| CRD | Field |
+| --- | ----- |
+| `Task` | `credentials.path` | 
+| `Task` | `spec.steps[].name` |
+| `Task` | `spec.steps[].image` |
+| `Task` | `spec.steps[].env.value` |
+| `Task` | `spec.steps[].env.valuefrom.secretkeyref.name` |
+| `Task` | `spec.steps[].env.valuefrom.secretkeyref.value` |
+| `Task` | `spec.steps[].env.valuefrom.configmapkeyref.name` |
+| `Task` | `spec.steps[].env.valuefrom.configmapkeyref.value` |
+| `Task` | `spec.steps[].volumemounts.name` |
+| `Task` | `spec.steps[].volumemounts.mountpath` |
+| `Task` | `spec.steps[].volumemounts.submountpath` |
```

Comment:
I think this should be `subpath`?

https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#volumemount-v1-core

https://github.com/tektoncd/pipeline/blob/master/pkg/apis/pipeline/v1beta1/container_replacements.go#L70